configure.ac

```diff
@@ -200,6 +200,7 @@ case "${host}" in
     AC_DEFINE(__FREEBSD__)
     AC_DEFINE(__BSD__)
     DEFAULT_DEFAULT_wxUSE_GTK=1
+    wxUSE_PROCSTAT=yes
```

Comment:
I don't think this is needed at all, we already check for `host` before testing for procstat below.